src/pages/successRequestResetPassword.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Card, Link, Stack, Typography } from "@mui/material";
+import MarkEmailReadIcon from "@mui/icons-material/MarkEmailRead";
+import { grey } from "@mui/material/colors";
+
+import Background from "@/components/public/Background";
+
+import { PagePaths } from "enum/pages";
+
+export default function Home() {
```

Comment:
I'm not sure that this is my section or not, but I asked and received answer in design review session that there is no restriction to access the page yet. If design team has already made decision on this issue, pls let me know.

Reply:
@ChariU What do you think about it?

Reply:
I’m a bit confused by what you asked. The `requestResetPassword` should access without login since its entry point is in login page via “Forget Password?” (I’m redesigning it) so the `successRequestResetPassword` should be access without login like the `requestResetPassword ` as I mentioned, but if you asked about whether this page should accessible without succeeding in `requestResetPassword` first or not, I would personally prefer no. Is this what you asked?

Reply:
Sorry, I give you an incorrect question. The correct one should be like Paper have said.

Reply:
So `successRequestResetPassword` page should be only accessible when `requestResetPassword` request is succeed, am I understand it correctly? Then, where should it route to when bypassing access occurs?

Reply:
May be both success pages should be popup instead of separated pages. I think it will be easier. Since user isn't logged in when they accessed these pages, we cant use user id to fetch or store any states in db. Another possibility is to use localStorage to pass some states through pages.